### DISCUSSION TITLE
Which older version still works on MacOS 14?

### DISCUSSION THREAD

**Discussion by yazze** [jeffvli/feishin#691] [Q&A]
Ditto.

**Comment by kgarner7** [comment]
v0.7.2 bumped the Electron version to 31, so the last version that will support macOS 10.13/10.14 is v0.7.1.

**Reply by jeffvli** [reply]
Adding onto this, macOS 10.13 / 10.14 support was [removed in Electron v27](https://www.electronjs.org/docs/latest/breaking-changes#planned-breaking-api-changes-270), which means you'll need to use a version of Electron v26 or older.

I'm not planning on supporting the older electron versions since v26 ended support in Feb 2024. I recommend keeping an up-to-date fork of this repository and building the binaries yourself so that you'll still have access to the latest changes.

You only need to make a few changes:

- Update the electron dependency version to a v26 version or older
https://github.com/jeffvli/feishin/blob/08db18359a84bdafff9ebe9fd01c3a684e9085a3/package.json#L234

- Update the `electronVersion` property in the electron-builder configuration to the version selected above
https://github.com/jeffvli/feishin/blob/08db18359a84bdafff9ebe9fd01c3a684e9085a3/package.json#L59

--- 

After making the above changes, you can either:
- Package the app locally using `npm install --legacy-peer-deps` and then `npm run package` 
- Package the app using the GitHub action `Publish macOS (Manual)` on the repo/fork and then download the build from the github release draft
